### ISSUE TITLE
Support "Simplified Tab Groups"

### ISSUE THREAD

**Issue by gene-pavlovsky** [piroor/multipletab#121]
I'm using the [Simplified Tab Groups](https://github.com/denschub/firefox-tabgroups) add-on and I can't see "Move to group" option when I select tabs. Do you have plans to add support for this add-on?
Or perhaps @denschub would like to chip in?

**Comment by piroor** [comment]
Currently I have no plan to do it by myself, but pull request for it is welcome.

**Comment by denschub** [comment]
Oh, sorry. At the moment, I am not able to work in this, since my tab groups extension is mostly dogfood and I am not using Multiple Tab Handler. I'm too busy at the moment to help you out, sorry. :/

**Comment by rayman89** [comment]
There are other add-ons that provide the same/similar functionality. Have you checked those to see if the move to group thing works?

**Comment by gene-pavlovsky** [comment]
@rayman89 no I didn't see other minimalist add-ons with this functionality, links?

**Comment by rayman89** [comment]
Don't wanna send you links because I don't use them but I've seen them. If they are exactly what you are looking for IDK but if you look for them you may find it.

https://addons.mozilla.org/en-US/firefox/search/?q=tab%20groups&cat=all

**Comment by gene-pavlovsky** [comment]
Ok I switched to the more mainstream [Tab Groups](https://addons.mozilla.org/en-US/firefox/addon/tab-groups-panorama/) add-on (ex-Panorama). Disabled the Simplified Tab Groups add-on first, then installed this one. It recognized the tab groups I already had without any special setup.

- I selected several tabs in one group, `Move to group` contained a list of my tab groups this time, I chose a group, but the tabs didn't disappear from the tab list of the current group. After a click on another tab one tab disappeared, but it didn't appear in the selected destination group. After I restarted Firefox, though, I saw a new auto-created group "Group 8" with just a single tab.
- I selected that tab, trying to move it to the original group it was in. However the "Move to group" (for any tab(s)) now lists only item, "New group".

Ideas?

**Comment by rayman89** [comment]
Then probably the instructions that were made for when it was a firefox feature are not compatible with the tab groups created by add-ons in general. If all the add-ons for groups worked the same then it could be requested as a feature if it will only work for one specific add-on I don't know if they will do it.  In the meantime you could use tree style tabs, tabkit2 or similar add-ons to group tabs into a category or to create folder tabs is not the same at all but it will help you organize tabs and you cam move them in bulk using MTH if I'm not mistaken.

**Comment by gene-pavlovsky** [comment]
Thanks for tips @rayman89, I was using TST before, but since I've decided to cure myself from many-tabs addiction (which it enabled me to keep), I've removed it. I went back to tab groups just to segregate my activities, e.g. keep personal tabs separate from work (and mostly don't open personal ones while working and vice versa). TST would show all of them in a tree and psychologically it's not as effective as tab groups which are well hidden unless I open their list.
